src/app/api/api_v1/endpoints/detections.py

```diff
@@ -4,9 +4,10 @@
 # See LICENSE or go to <https://www.apache.org/licenses/LICENSE-2.0> for full license details.
 
 import hashlib
+import logging
 from datetime import datetime
 from mimetypes import guess_extension
-from typing import List, cast
+from typing import List, Optional, cast
```

Comment:
Oh, with the next version of python, it's been widely discussed in the community, but basically, we have to change:
```python
Optional[TypeABC]
 ```
into:
```python
Union[TypeABC, None]
```